src/lib.rs

```diff
@@ -149,6 +150,37 @@ fn enforce_supplemental_groups(
     Ok(None)
 }
 
+fn enforce_container_security_policies(
+    container: &mut apicore::Container,
+    validation_request: &ValidationRequest<Settings>,
+) -> Result<bool> {
+    let mut mutated: bool = false;
+    match enforce_run_as_user_rule(container.security_context.clone(), validation_request) {
+        Ok(mutate_request) => {
+            if mutate_request.is_some() {
+                mutated = true;
+                container.security_context = mutate_request;
+            }
+        }
+        Err(error) => {
+            return Err(error);
+        }
+    }
+
+    match enforce_run_as_group(container.security_context.clone(), validation_request) {
```

Comment:
A similar refactoring can be done here too. The usage of `?` make the code more concise